templates/web/base/main_nav_items.html

```diff
@@ -1,4 +1,4 @@
-[%~ INCLUDE navitem uri='/' label=loc('Report a problem') attrs='class="report-a-problem-btn"' ~%]
+[%~ INCLUDE navitem uri=(homepage_uri OR '/') label=loc('Report a problem') attrs='class="report-a-problem-btn"' ~%]
```

Comment:
`homepage_uri` isn't always set, so e.g. on `/auth` the 'Report a problem' link is to `/`, not `/report`. This seems to be a bug, unless I'm misunderstanding something perhaps.